Automattic/wp-calypso
# Add P2 autocomplete to block editor

#### Changes proposed in this Pull Request

~Needs D32779-code.~ _(Deployed)_

See p6OVho-fw-p2 for the original project proposal, including mockups illustrating the feature.

#### Testing instructions

_Apply D32832-code to your sandbox, and sandbox a P2. See that diff's description for more detailed instructions._

D32832-code generated by doing the following:

- Find the `ci/circleci: build-o2-blocks` check at the bottom of this PR and click on "Details" to find the job number.
- Log in to your sandbox and fetch the updated block code with the bin script specified in #gm-19-project-p2-auto-suggestions
